docs/sdk/src/guides/enable_elastic_scaling.rs

```diff
@@ -0,0 +1,162 @@
+//!
+//! ## Dependencies
+//!
+//! Prerequisites: Polkadot-SDK `2509` or newer.
+//!
+//! To ensure the security and reliability of your chain when using this feature you need the
+//! following:
+//! - An omni-node based collator. This has already become the default choice for collators.
+//! - [RFC103](https://github.com/polkadot-fellows/RFCs/blob/main/text/0103-introduce-core-index-commitment.md).
+//!   This is mandatory protection against PoV replay attacks.
+//! - Enabling the relay parent offset feature. This is required to ensure the parachain block times
+//!   and transaction in-block confidence are not negatively affected by relay chain forks.
+//! - Block production configuration adjustments.
+//!
+//! ### Upgrade to Polkadot Omni node
+//!
+//! Your collators need to run `polkadot-parachain` or `polkadot-omni-node` with the `--authoring
+//! slot-based` CLI argument.
+//!
+//! Further information about omni-node and how to upgrade is available:
+//! - [high level docs](https://docs.polkadot.com/develop/toolkit/parachains/polkadot-omni-node/)
+//! - [`crate::reference_docs::omni_node`]
+//!
+//! ### Enable RFC103
+//!
+//! RFC103 is enabled automatically on the collators if it is enabled on the relay chain. There are
+//! no code changes required on the client to support it.
+//!
+//! RFC103 relies on the ability of parachain blocks to commit to a specific core index on the relay
+//! chain. This commitment is implemented via `UMP` signals, which rely on the upward message
+//! passing channel that is used by parachains to send messages to the relay chain.
+//!
+//! The only required change for the runtime is enabling the `experimental-ump-signals` feature of
+//! the `parachain-system` pallet:
+//! `cumulus-pallet-parachain-system = { workspace = true, features = ["experimental-ump-signals"]
+//! }`
+//!
+//! You can find more technical details about RFC103 [here](https://github.com/polkadot-fellows/RFCs/blob/main/text/0103-introduce-core-index-commitment.md).
+//!
+//! ### Enable the relay parent offset feature
+//!
+//! It is recommended to use an offset of `1`, which is sufficient to eliminate any issues
+//! with relay chain forks.
+//!
+//! Configure the relay parent offset like this:
+//! ```ignore
+//!     /// Build with an offset of 1 behind the relay chain best block.
+//!     const RELAY_PARENT_OFFSET: u32 = 1;
+//!
+//!     impl cumulus_pallet_parachain_system::Config for Runtime {
+//!         // ...
+//!         type RelayParentOffset = ConstU32<RELAY_PARENT_OFFSET>;
+//!     }
+//! ```
+//!
+//! Implement the runtime API to retrieve the offset on the client side.
+//! ```ignore
+//!     impl cumulus_primitives_core::RelayParentOffsetApi<Block> for Runtime {
+//!         fn relay_parent_offset() -> u32 {
+//!             RELAY_PARENT_OFFSET
+//!         }
+//!     }
+//! ```
+//!
+//! ### Block production configuration
+//!
+//! This configuration directly controls the minimum block time and maximum number of cores
```

Comment:
should we add a note saying that they don't need to make changes to the slot duration? Parachains that were using the MVP may be used to modifying it
And a recommendation for keeping it at 6seconds, or is it 12?